ci_launcher.py

```diff
@@ -48,6 +48,15 @@ def launch(self):
             if not self._tests_config[board]['tests']:
                 logging.info("  No test set")
 
+            # Retrieve the device status
+            dev = self.crafter.get_device_status(board)
+            if dev['status'] == "offline":
+                logging.error("Device is offline, not sending jobs")
+                continue
+            elif dev['status'] == "retired":
+                logging.error("Device is retired, not sending jobs")
+                continue
+
```

Comment:
If you don't mind squashing the two really similar log messages, you can write something like this in Python:  
`if dev['status'] in ['offline', 'retired']: ...`

And if the different error messages are really that important:  
`logging.error("Device is %s, not sending jobs" % dev['status'])`